### DISCUSSION TITLE
Should we add `vim.loader.enable()` to init.lua

### DISCUSSION THREAD

**Discussion by Saafo** [ayamir/nvimdots#715] [Ideas]
nvim 0.9 has a [new lua module loader](https://github.com/neovim/neovim/pull/22668) to speed up startup time, but I'm not sure whether it helps since we're already using lazy.nvim.

**Comment by Jint-lzxy** [comment]
It's already there: [`lazy/core/cache.lua`](https://github.com/folke/lazy.nvim/blob/main/lua/lazy/core/cache.lua).
